src/components/menu/menu.stories.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+  import { Meta, Story, Template } from '@storybook/addon-svelte-csf'
+
+  import Menu from './menu.svelte'
+  import Icon from '../icon/icon.svelte'
+
+  let button
+</script>
+
+<Meta title="Menu" component={Menu} />
+
+<Template let:args>
+  <div class="container">
+    <button bind:this={button}> View page source </button>
```

Comment:
I think let's just remove this button - not really needed for the`Menu` component

Reply:
removed